ui/sections/storage/inspectors/snapshot.reel/snapshot.js

```diff
@@ -37,6 +37,21 @@ exports.Snapshot = AbstractInspector.specialize(/** @lends Snapshot# */ {
         value: null
     },
 
+    _recursive: {
+        value: null
+    },
+
+    recursive: {
```

Comment:
This value should probably be reset in `exitDocument` or added as a property of `object` to avoid keeping the state between multiple snapshot creation.